catalog-info.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: sci-react-ui
+  title: Scientific React UI
+  description: Scientific React UI components, based on MUI. Early development release. A theme and component library to make websites at scientific institutions simple to create.
```

Maybe we should drop the early development release (are we a year old now?!)
```suggestion
  description: Scientific React UI components, based on MUI. A theme and component library to make websites at scientific institutions simple to create.
```